hoot-core/src/main/cpp/hoot/core/algorithms/ProbabilityOfMatch.cpp

```diff
@@ -110,7 +110,7 @@ double ProbabilityOfMatch::distanceScore(const ConstOsmMapPtr& map, const ConstW
     Point* point(GeometryFactory::getDefaultInstance()->createPoint(v[i]));
     if (debug)
     {
-      cout << "distance " << ls2->distance(point) << endl;
+      LOG_INFO("distance " << ls2->distance(point));
```

Comment:
Replaced `std::cout` with `LOG_*` and `PROGRESS_*` based on the logging level requested.  This one was `INFO` to match code below.